### PR TITLE
Add namespacing support to pkgdb-cli and pkgdb-admin

### PR DESCRIPTION
**Pull request by pypingou**
This should however remain backward compatible both for the UI interface and the library.

### CODE REVIEW

File: pkgdb2client/cli.py [hunk 1]
```diff
@@ -51,6 +51,18 @@ class ActionError(Exception):
     pass
 
 
+def __get_namespace(args):
+    ''' Return the namespace of the package as specified as an argument
+    unless it was specified in the package name.
+    '''
+    namespace = args.namespace
+    if hasattr(args, 'package'):
+        print '***', args.package
```
Comment (ralphbean) [inline pkgdb2client/cli.py]: Debug statement?

Comment (pypingou) [inline pkgdb2client/cli.py]: def :)

**Comment by ralphbean** [comment]
Looks good here.

**Comment by pypingou** [comment]
Let's merge then :)

**Pull request merged**